src/main/java/com/google/devtools/build/lib/bazel/repository/RepositoryOptions.java

```diff
@@ -159,6 +159,14 @@ public class RepositoryOptions extends OptionsBase {
               + " defined.")
   public Duration httpConnectorRetryMaxTimeout;
 
+  @Option(
+      name = "experimental_http_max_parallel_downloads",
```

Comment:
I would suggest to drop the `experimental_` prefix. This is mostly used to gate new features with logic that may not be production ready, but here we are just exposing a single new parameter.